scripts/fintual.js

```diff
@@ -0,0 +1,82 @@
+// Description:
+//   Obtiene valores de los distintos fondos de la AGF Fintual
+//
+//
+// Commands:
+//   hubot fintual help - Imprime la ayuda
+//   hubot fintual risky norris|moderate pitt|conservative clooney|conservative streep - Obtiene el valor del fondo seleccionado
+//
+// Author:
+//   @cvallejos
+
+const CLP = require('numbertoclpformater').numberToCLPFormater
+const FINTUAL_REAL_ASSETS_API_URL = process.env.API_URL || 'https://fintual.cl/api/real_assets/'
+const NORMALIZE_AMOUNT = 100000
+var NORRIS_A = 'risky norris'
+var NORRIS_APV = 'norris-apv'
+var PITT_A = 'moderate pitt'
+var PITT_APV = 'pitt-apv'
+var CLOONEY_A = 'conservative clooney'
+var CLOONEY_APV = 'clooney-apv'
+var STREEP_A = 'conservative streep'
+
+const series = []
```

Comment:
Para esta variable conviene usar mejor un Map
```suggestion
const series = new Map([
  ['risky norris', 186],
  ['norris-apv', 245],
  ['moderate pitt', 187],
  ['pitt-apv', 246],
  ['conservative clooney', 188],
  ['clooney-apv', 247],
  ['conservative streep', 15077],
])
const comands = Array.from(series.keys())
```